### ISSUE TITLE
Featured posts on front page

### ISSUE THREAD

**Issue by chrisdubia** [oeco/jeo#79]
Hi Gustavo,

How can we make the containers for the featured posts on the homepage the same size and is it possible to include a thumbnail in the preview window?

best
chris

**Comment by tamielbr** [comment]
do you intend to do thumbnails like here, on each window (http://mamu.net.br/)?

**Comment by chrisdubia** [comment]
Exactly!  That would be great.  Is there also a way to make the containers
of the preview excerpts the same size?
The example you shared also have different sizes.

Thanks so much!

On Thu, Feb 25, 2016 at 7:21 AM, Tel notifications@github.com wrote:

> do you intend to do thumbnails like here, on each window (
> http://mamu.net.br/)?
> 
> —
> Reply to this email directly or view it on GitHub
> https://github.com/oeco/jeo/issues/79#issuecomment-188758734.

## 

[image: Chris-Signature.png]

**Comment by tamielbr** [comment]
You have to edit `loop.php` (or better, create a child theme) and under:

`<header class="post-header"><h3>` 

add this to create the thumbnail:
`<div align="center"><?php`
`if ( has_post_thumbnail() ) { // check if the post has a Post Thumbnail assigned to it.`
`the_post_thumbnail( 'thumbnail' );`
`} ?> </div>`

i haven't tried making them the same size, but i would guess it is controled by `<?php the_excerpt(); ?>` which puts out a maximum text size, which you might be able to control via the function; but there might be other ways via CSS.

**Comment by chrisdubia** [comment]
Thanks very much for your quick response and help. It worked!

Question for you.  We are a very small nonprofit and have a limited
capacity for tech development and are in need of someone that can help us
customize our site.  Not very much to do - just a few lines of code to
customize the jeo theme

thoughts?
best
c

On Thu, Feb 25, 2016 at 7:43 AM, Tel notifications@github.com wrote:

> You have to edit loop.php (or better, create a child theme) and under:
> 
> <header class="post-header"><h3>
> 
> add this to create the thumbnail:
> 
> <div align="center"><?php
> if ( has_post_thumbnail() ) { // check if the post has a Post Thumbnail
> assigned to it.
> the_post_thumbnail( 'thumbnail' );
> } ?> </div>
> 
> i haven't tried making them the same size, but i would guess it is
> controled by <?php the_excerpt(); ?> which puts out a maximum text size,
> which you might be able to control via the function; but there might be
> other ways via CSS.
> 
> —
> Reply to this email directly or view it on GitHub
> https://github.com/oeco/jeo/issues/79#issuecomment-188771167.

## 

[image: Chris-Signature.png]

**Comment by tamielbr** [comment]
@cardume @miguelpeixe perhaps can help?

**Comment by chrisdubia** [comment]
I'll send him a note.  Thanks!
